File: ICP/Assignment_7/Q8.java
/*Name-Anish More
  Registration number:2241013365
  Name of the program:Write a java program that implements the array reversal algorithm suggested in Note 1.
Note 1: There is a simpler algorithm for array reversal that starts out with two indices, i=0
and j=n-1. With each iteration i is increased and j is decreased for i<j.*/
import java.util.Scanner;
public class Q8
{
	public static void main(String[] args) 
	{
		Scanner sc=new Scanner(System.in);
		System.out.print("Enter number of terms in array:");
		int n=sc.nextInt();
		int arr[]=new int[n];
		for(int i=0;i<n;i++)
		{
			arr[i]=sc.nextInt();
		}
		int i=0,j=n-1;
		while(i<j)
		{
			int temp=arr[i];
			arr[i]=arr[j];
			arr[j]=temp;
			i++;
			j--;
		}
		for(int k=0;k<n;k++)
		{
			System.out.print(arr[k]+"   ");
		}
	}
}
	
/*Output
Enter number of terms in array:5
2
3
1
5
6
6   5   1   3   2   
 */

File: Python/Minor_Assignment_3/Q8.py
def days(n):
    if n in [1,3,5,7,8,10,12]:
        print("No of days: 31 days")
    elif n in [4,6,9,11]:
        print("No of days: 30 days")
    else:
        print("No of days: 28/29 days")

n=int(input("Enter a month number: "))
days(n)

'''
Output--
Enter a month number: 4
No of days: 30 days
'''
        